src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -189,6 +194,20 @@ export const ChooseServicePage = () => {
     }
   };
 
+  const onCancel = () => {
```

Comment:
Should this method have been made into a reusable one? (under a different name)

Reply:
Tenkte på det selv men så på de andre altinn2 URL-ene og de varierer så mye at man ikke får abstrahert det (som jeg kan se i alle fall). I tillegg tror jeg det bare vil være på denne Pagen at vi sender brukeren til denne urlen per nå